client/apps/game/src/ui/modules/entity-details/hex-entity-details.tsx

```
@@ -0,0 +1,105 @@
+
+    return getComponentValue(
+      dojo.setup.components.Tile,
+      getEntityIdFromKeys([BigInt(selectedHexContract.x), BigInt(selectedHexContract.y)]),
+    );
+  }, [selectedHex?.col, selectedHex?.row]);
+
+  const userAddress = ContractAddress(dojo.account.account.address);
+
+  let entityContent = null;
+
+  const hasOccupier = tile && tile.occupier_id;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)